Calendar/BookingHelpers/GraphInstaller.ps1

```diff
@@ -0,0 +1,89 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# ===================================================================================================
+# Graph Modules installer Functions
+# ===================================================================================================
+function CheckGraphBookingsModuleInstalled {
```

Comment:
>CheckGraphBookingsModuleInstalled [](http://example.com/codeflow?start=9&length=33)

Future work:
When we move this to Shared, we should pass in the Module to the "CheckIfModuleIsInstalled" as one string is all that looks to be different in these 3 functions (these 2 + ImportExcel)

Reply:
We will need to refactor our code to use the shared function from the Load-Module.ps1 in the .build directory.
This function already does the work we want, although it has additional parameters